ipa_room_segmentation/ros/launch/room_segmentation_action_server.launch

```diff
@@ -3,7 +3,7 @@
 
 
 	<!--  -->
-	<node ns="room_segmentation" pkg="ipa_room_segmentation" type="room_segmentation_server" name="room_segmentation_server" output="screen" respawn="true" respawn_delay="2">
+	<node ns="room_segmentation" pkg="ipa_room_segmentation" type="room_segmentation_server" name="room_segmentation_server" output="screen" respawn="false" respawn_delay="2">
```

Comment:
Why false? Wasn't this our fallback for sporadic crashes of the component? Or does it never crash anymore?

Reply:
True, overlooked that during the push, thanks for notifying. Reverted it to `respawn=true` in the new commit.